pkg/vcsigner/signer_test.go

```diff
@@ -48,6 +52,17 @@ func TestSigner_New(t *testing.T) {
 	})
 }
 
+func getLoader(t *testing.T) *jsonld.DocumentLoader {
+	t.Helper()
+
+	loader, err := jsonld.NewDocumentLoader(mem.NewProvider(),
+		jsonld.WithRemoteDocumentLoader(ld.NewDefaultDocumentLoader(&http.Client{})),
```

Comment:
We shouldn't be doing remote loader. Preload the orb contexts.

Not even in tests.